app/Controllers/Movies/Home.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace App\Controllers\movies;
+use CodeIgniter\RESTful\ResourcePresenter;
+
+class Home extends ResourcePresenter
```

Comment:
Your namespace and filename do not match.
Further, I see another file below app/Controllers/movies/Home.php???
This is confusing and should be fixed.